book/src/ephemeral-local-volumes.md

```diff
@@ -116,6 +118,17 @@ supports dynamic provisioning of volumes. No other changes are needed
 in the driver in that case. Such a driver can also support CSI
 ephemeral inline volumes if desired.
 
+## Security Considerations
+
+CSI driver vendors that choose to support ephemeral inline volumes are responsible for secure handling of these volumes, and special consideration needs to be given to what volumeAttributes are supported by the driver. As noted above, a CSI driver is not suitable for CSI ephemeral inline volumes when volume creation requires volumeAttributes that should be restricted to an administrator. These attributes are set directly in the Pod spec, and therefore are not automatically restricted to an administrator when used as an inline volume.
+
+CSI inline volumes are only intended to be used for ephemeral storage, and driver vendors should NOT allow usage of inline volumes for persistent storage unless they also provide a third party pod admission plugin to restrict usage of these volumes.
+
+Cluster administrators who need to restrict the CSI drivers that are
+allowed to be used as inline volumes within a Pod spec may do so by:
+- removing `Ephemeral` from `volumeLifecycleModes` in the CSIDriver spec
```

Comment:
Can you change this to be the same as what you have in the K8s website doc PR?
```
Removing `Ephemeral` from `volumeLifecycleModes` in the CSIDriver spec, which prevents the driver from being used as an inline ephemeral volume.
```

Reply:
Updated, thanks!